nc-games/src/components/ListedReview.jsx

```diff
@@ -1,41 +1,46 @@
+import { Link } from "react-router-dom";
+
 const ListedReview = ({ review }) => {
   const dateString = `${review.created_at.slice(
     8,
     10
   )}-${review.created_at.slice(5, 7)}-${review.created_at.slice(0, 4)}`;
+
   const timeString = review.created_at.slice(11, 16);
 
   return (
-    <li>
-      <h3>{review.title}</h3>
-      <label htmlFor="designed-by" className="label">
-        Designed By
-      </label>
-      <label htmlFor="owner" className="label">
-        Review By
-      </label>
-      <p className="designer">{review.designer}</p>
-      <p className="owner">{review.owner}</p>
-      <img
-        className="review-img"
-        alt={`image for the review ${review.title}`}
-        src={review.review_img_url}
-      />
-      <label htmlFor="comments" className="label bottom">
-        Comments
-      </label>
-      <label htmlFor="date" className="label bottom">
-        Posted At
-      </label>
-      <label htmlFor="votes" className="label bottom">
-        Votes
-      </label>
-      <p className="bottom">{review.comment_count}</p>{" "}
-      <p className="bottom">
-        {timeString} {dateString}
-      </p>
-      <p className="bottom">{review.votes}</p>
-    </li>
+    <Link to={`/review/${review.review_id}`}>
+      <li>
+        <h3>{review.title}</h3>
+        <label htmlFor="designed-by" className="label">
```

Comment:
One could argue that if you're giving a label a className of 'label', you could just access it by tag type in your css (by just using "label" instead of ".label"). Never typed label so many times in one message before.